### DISCUSSION TITLE
Please help me to find them | Pending open channel

### DISCUSSION THREAD

**Discussion by Tapao-NonSen** [Ride-The-Lightning/RTL#923] [Troubleshooting Questions] [answered]
I am stuck on finding the output index and fee to open channel in pending
Bump fee

![](https://i.imgur.com/mFn2aN5.jpg)

**Comment by saubyk** [comment, marked as answer]
Hello @Yusaki-Akito I am assuming the channel has been confirmed now.
Couple of hints in case you need to do this again in the future:
- Typically the change to the channel open transaction would have an output index value of either 1 or 0. So if 1 is not working you can try 0. Ideally, you should confirm this though by inspecting the transaction in a block explorer.
- As explained in the instructions, you should inspect your transaction in a block explorer (ideally running on your own node), by searching for the transaction ID, which is available on this window
- The most important thing to understand here is, do you even have enough change output to bump the transaction. If the output is a smaller value and mempool is busy enough, you may not have an option to bump up the channel open this way
- Also I see that you entered a value or `1.007` to bump. This value is typically not enough to bump the channel open, if the mempool is really busy. You need to look at the mempool.space site/app to determine the optimal rate for the change output transaction.

Hope this helps. Thanks.
